src/plugins/pcr_primer_design_for_dna_assembly/src/tasks/PCRPrimerDesignForDNAAssemblyTask.cpp

```diff
@@ -43,6 +45,244 @@
 
 namespace U2 {
 
+//To generate a PCRPrimerDesignForDNAAssemblyTask report.
+namespace ReportUtils {
```

Comment:
Look, you have a new namespace here, but it's used just in one place by `using namespace ReportUtils;`. It looks a bit over-designed. I suggest you to get rid of this namespace